src/preprocessing_DataReader.py

```diff
@@ -69,24 +110,40 @@ def load(self, patient: Union[int|List[int]], experiment: Union[int|List[int]])
                     file_path = os.path.join(patient_path, file)
                     patient_data = self._read_edf_file(file_path)
                     self.edf_data = pd.concat([self.edf_data, patient_data], ignore_index=True)
-        return self.edf_data
+
 
     def normalize(self, norm_type: str = "min-max"):
```

Comment:
```suggestion
    def normalize(self, norm_type: Literal['min-max', 'z-score'] = "min-max") -> None:
```

note: requires `from typing import Literal`

Reply:
Done